unity-client/Assets/Scripts/MainScripts/DCL/Controllers/Settings/Settings.cs

```diff
@@ -76,12 +91,18 @@ private void LoadGeneralSettings()
             }
         }
 
-        public void ApplyQualitySettingsPreset(int index)
+        public void ApplyAutoQualitySettings(int index)
```

Comment:
Only as suggestion: As a public function that will be called from other entitie(s), it would be fine to add a header with some general documentation about what this function does.